src/pshtt/pshtt.py

```diff
@@ -1562,7 +1562,7 @@ def load_preload_list():
     utils.debug("Fetching Chrome preload list from source...", divider=True)
 
     # Downloads the chromium preloaded domain list and sets it to a global set
-    file_url = "https://chromium.googlesource.com/chromium/src/net/+/master/http/transport_security_state_static.json?format=TEXT"
+    file_url = "https://chromium.googlesource.com/chromium/src/net/+/main/http/transport_security_state_static.json?format=TEXT"
```

Comment:
Manually navigating the Chromium source repository has this URL (that I added the format argument to for the final suggested URL). The one you used works fine but I thought I would mention this as a suggestion.

```suggestion
    file_url = "https://chromium.googlesource.com/chromium/src/+/main/net/http/transport_security_state_static.json?format=TEXT"
```

Reply:
I think I like my original change better, since it only updates the branch name.  I think that is the only thing that changed upstream.

Reply:
Alright. I just figured it was worth mentioning since the URL has been unchanged since it was added over six years ago with no context around how it was determined. I wasn't clear on why the branch was after the first directory of the object but I'm not super familiar with gitilies. This is just given that the same file is accessed at `https://github.com/chromium/chromium/blob/main/net/http/transport_security_state_static.json` on the GitHub mirror.

Reply:
That line was added before I worked on `pshtt`.  [@KyleEvers might remember where the original URL came from](https://github.com/cisagov/pshtt/blame/b6e89f9756fb5548db0c53afe40de746cc78eb2f/site_inspector.py#L326).